test/common/protobuf/utility_test.cc

```diff
@@ -216,11 +216,10 @@ TEST_F(ProtobufUtilityTest, RepeatedPtrUtilDebugString) {
   Protobuf::RepeatedPtrField<ProtobufWkt::UInt32Value> repeated;
   EXPECT_EQ("[]", RepeatedPtrUtil::debugString(repeated));
   repeated.Add()->set_value(10);
-  EXPECT_THAT(RepeatedPtrUtil::debugString(repeated),
-              testing::ContainsRegex("\\[.*[\n]*value:\\s*10\n\\]"));
+  EXPECT_THAT(RepeatedPtrUtil::debugString(repeated), testing::ContainsRegex("\\[value: 10\n\\]"));
   repeated.Add()->set_value(20);
   EXPECT_THAT(RepeatedPtrUtil::debugString(repeated),
-              testing::ContainsRegex("\\[.*[\n]*value:\\s*10\n,.*[\n]*value:\\s*20\n\\]"));
+              testing::ContainsRegex("\\[value: 10\n, value: 20\n\\]"));
```

Comment:
does this test work without this expectation change?

Reply:
at least cannot work at my local env. I can revert it to see if our ci say ok.

Reply:
fine, our CI say it ok